backend/users/api/serializers.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import fields
+from rest_framework import serializers
+
+from users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    password = serializers.CharField(write_only=True)
+
+    class Meta:
+        model = User
+        fields = "__all__"
+
+        
```

Comment:
remove extra spaces then add new line always at end of file, pls also apply this to other files